apps/src/studio/studio.js

```diff
@@ -1578,8 +1579,9 @@ Studio.clearEventHandlersKillTickLoop = function() {
   });
   Studio.eventHandlers = [];
   Studio.perExecutionTimeouts.forEach(function (timeout) {
-    clearInterval(timeout);
+    clearTimeout(timeout);
   });
+  clearInterval(Studio.tickIntervalId);
```

Comment:
Sinon revealed an issue where we were mixing setTimeout with clearInterval. I changed things so that perExecutionTimeouts is all setTimeout/clearTimeout, and then onTick uses clearInterval.